docs/getting-started/local-setup/setting-up-a-local-chain.md

```diff
@@ -13,7 +13,7 @@ Additionally, you will see the different options you can use to make your Gno in
 
 - [`gnoland` installed](local-setup.md#3-installing-other-gno-tools).
```

Comment:
We're moving away from having `gnoland` installed in the Getting started section. Please add a short "Installation" section instead of this

Reply:
This is not added in this PR -- please change it in a separate one, or open an issue